addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -362,18 +362,18 @@ pscanrules.usernameidor.soln = Use per user or session indirect object reference
 
 pscanrules.viewstate.content.email.desc = The following emails were found being serialized in the viewstate field:
 pscanrules.viewstate.content.email.name = Emails Found in the Viewstate
-pscanrules.viewstate.content.email.pattern.source = Email pattern - http://www.regular-expressions.info/regexbuddy/email.html
+pscanrules.viewstate.content.email.pattern.source = Email pattern - https://www.regular-expressions.info/examples.html
 pscanrules.viewstate.content.ip.desc = The following potential IP addresses were found being serialized in the viewstate field:
 pscanrules.viewstate.content.ip.name = Potential IP Addresses Found in the Viewstate
-pscanrules.viewstate.content.ip.pattern.source = IP pattern - http://www.regular-expressions.info/examples.html
+pscanrules.viewstate.content.ip.pattern.source = IP pattern - https://www.regular-expressions.info/ip.html
 pscanrules.viewstate.name = Viewstate
 pscanrules.viewstate.nomac.sure.desc = This website uses ASP.NET's Viewstate but without any MAC.\n\n
 pscanrules.viewstate.nomac.sure.name = Viewstate without MAC Signature (Sure)
-pscanrules.viewstate.nomac.sure.refs = http://msdn.microsoft.com/en-us/library/ff649308.aspx
+pscanrules.viewstate.nomac.sure.refs = https://msdn.microsoft.com/en-us/library/ff649308.aspx
 pscanrules.viewstate.nomac.sure.soln = Ensure the MAC is set for all pages on this website.
 pscanrules.viewstate.nomac.unsure.desc = This website uses ASP.NET's Viewstate but maybe without any MAC.\n\n
 pscanrules.viewstate.nomac.unsure.name = Viewstate without MAC Signature (Unsure)
-pscanrules.viewstate.nomac.unsure.refs = http://msdn.microsoft.com/en-us/library/ff649308.aspx
+pscanrules.viewstate.nomac.unsure.refs = https://msdn.microsoft.com/en-us/library/ff649308.aspx
```

Comment:
While these do work they're also redirects, the may as well link the final destination with `)?redirectedfrom=MSDN` removed.

Reply:
ok